docs/docsource/operation/account_operations/manage_access_keys.rst

```diff
@@ -0,0 +1,49 @@
+.. _manage_access_keys:
+
+Manage Access Keys
+==================
+
+Creating Access Keys
+--------------------
+
+#. Log into the |product| UI using previously provisioned credentials (refer to :ref:`Create an Account`).
+
+   .. image:: ../../graphics/sign_into_artesca.PNG
+
+#. Click **Accounts** on the top menu.
+
+   .. image:: ../../graphics/account_banner.PNG
+
+#. Click **+Create Access key**
+
+   .. image:: ../../graphics/accounts-page.PNG
+
+#. Click **Continue** on the **Create Root user Access keys** screen.
+
+   .. image:: ../../graphics/root_user_access_keys_dialog.PNG
+
+#. Save the Secret Access Key ID by clicking the copy button, and save it in a secure location 
+   
+   .. image:: ../../graphics/close_button.PNG
+
+#. Close **Close**.
+
+Deleting Access Keys
+--------------------
+
+#. Log into the |product| UI.
+
+   .. image:: ../../graphics/sign_into_artesca.PNG
+
+#. Click **Accounts** on the top menu.
+
+   .. image:: ../../graphics/account_banner.PNG
+
+#. Click the red trash icon to the right of the access key you wish to delete. 
+
+   .. note::
```

Comment:
Maybe put this in an important. I feel like a note isn't enough.

Reply:
I agree